change/@microsoft-teams-js-aa74624b-2248-43c6-a090-d25c5c0ad877.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Add documentation for how to enable logging",
```

Comment:
Please use past tense for change files for consistency. (I'll add this to the CONTRIBUTING file in the change file section.)

Reply:
👍